### DISCUSSION TITLE
Troubleshooting geemap - layers not showing

### DISCUSSION THREAD

**Discussion by erica-mccormick** [datapane/datapane#148] [Help] [answered]
Hello,
I have several pages up which are composed of maps made using the geemap package with layers from Google Earth Engine. The basemaps and layout of the page are fine, however as of a few days ago, the layers from Google Earth Engine are not appearing. Any ideas?

Thank you so much! These are my first-ever pages with datapane so I am very new, but not new at all to Google Earth Engne or python.

**Comment by johnmicahreid** [comment, marked as answer]
Hi Erica, 

Thanks for sharing this - would you mind sharing a code snippet so we can reproduce the issue? 

Based on the description so far, I think it may be a geemap rather than Datapane issue - is the map appearing correctly inside your local notebook?
